pages/_app.js

```diff
@@ -128,20 +138,27 @@ export default function App({ Component, pageProps }) {
   }
 
   function handleHasCompletedStep4() {
-    alert("Labels for the x-axis and the y-axis of the graph are assigned");
+    alert("Labels for the x-axis and the y-axis of the chart are assigned.");
     setHasCompletedStep4(true);
   }
 
   function handleHasCompletedStep5() {
-    alert("Title of the graph is assigned");
+    alert("Title of the chart is assigned.");
     setHasCompletedStep5(true);
   }
 
+  function handleHasCompletedDistProp() {
```

Comment:
Also the function name would be more readable without shorthand notion

Reply:
We have renamed this function as handleHasCompletedDistributionProperties.